### DISCUSSION TITLE
Unable to run Phrozen Sonic Mini 8K Exposure time finder files

### DISCUSSION THREAD

**Discussion by Albert07LR** [sn4k3/UVtools#430] [Q&A] [answered]
Hi. I know Phrozen has just released its Sonic Mini 8K, but any chance someone knows how to make the "Exposure time finder" work in this printer? CTB files (versions 1 to 4) didn't work and CTB (encrypted) files show this message: "Different resolution with projector! - 1440". I assume this is the number of pixels of the LCD screen but I have no idea of how to change it.  Help would be much appreciated.

**Comment by sn4k3** [comment, marked as answer]
Phrozen Sonic Mini 8K uses the encrypted format which you can't save under UVtools.
But what slicer you use? You should be able to open the file on UVtools generate the test and convert to ctb encrypted to save it

**Reply by Albert07LR** [reply]
Thanks a lot for the quick answer! My bad. I just realised this machine only works with CTB files from Chitubox slicer. It worked perfectly fine the way you described it

**Comment by adsSintra** [comment]
when you say convert to ctb encrypted by this you mean open the file from uv tools in chitubox and save it again?

**Reply by sn4k3** [reply]
This is no longer valid on current version. 
You just need to open one file you sliced before and generate the test.
